README.md

```diff
@@ -21,83 +21,83 @@ Sensitive Tree.
 
 To achieve the first goal, the primary output of this repository is a
 static website which allows the reader to understand the definitions
-contained here, and link to other online resources where more information
+contained there and link to other online resources where more information
```

Comment:
shouldn't this still be `here` since it is referring to this repo, which contains the language definitions?

Reply:
I interpreted `static website` as the "here" and made the change, but I see now that you're correct.